metaci/build/models.py

```diff
@@ -325,6 +326,11 @@ def run(self):
             self.flush_log()
             return
 
+        try:
+            self.org_api_version = org_config.latest_api_version
+        except BaseException as e:
```

Comment:
I'll approve this, but Exception is probably a better choice than BaseException generally...if the user triggers a KeyboardInterrupt or something raises SystemExit we generally don't want to swallow that.

Reply:
Okay I updated it and made another super-minor change.